packages/core/webpack.config.js

```diff
@@ -62,6 +62,11 @@ module.exports = {
           to: './',
           context: path.resolve(__dirname, '../', 'web-components/dist'),
         },
+        {
+          from: '*',
+          to: './components/types',
+          context: path.resolve(__dirname, '../', 'web-components/dist/types'),
+        },
```

Comment:
I don't fully understand how the team that is needing this will be using it so did we need to double check that this is a good location for them?

I also noticed this comment in the original ticket description that I'm not sure is relevant or not:

> We also may need to make sure that the core package has these type definition files and/or has a types property in the package.json.

Reply:
I added that to the package.json file.